documentation/tutorials/getting-started-c/corelib.md

```diff
@@ -53,7 +53,7 @@ void main(void) {
     SCREEN *screen;
 +   load_library("/lib/core");
     get_lcd_lock();
-    screen = screen_allocate();
```

Comment:
Alright. I expect that might confuse some people but we'll see how many people ask in the IRC channel.